c2corg_ui/static/js/authservice.js

```diff
@@ -0,0 +1,181 @@
+goog.provide('app.Authentication');
+
+goog.require('app');
+
+
+
+/**
+ * @param {string} apiUrl URL to the API.
+ * @constructor
+ */
+app.Authentication = function(apiUrl) {
+
+  /**
+   * @type {string}
+   * @private
+   */
+  this.apiUrl_ = apiUrl;
+
+  /**
+   * @type {?Object}
+   * @private
+   */
+  this.userData_ = null;
+};
+
+
+/**
+ * @return {boolean}
+ * @export
+ */
+app.Authentication.prototype.isAuth = function() {
```

Comment:
> Well, since exported functions are not shortnamed by closure, I wanted to use a short name.

Don't worry about this. Internally a "shortnamed" name will be used and only at the end there will be something like "app.Authentication.prototype.isAuth = Xy.fo".

Additionally the html and js files will be gzipped, so there is no need to use shorter variable or function names! :)